Example/Controllers/PointsSettingsTableViewController.swift

```diff
@@ -50,4 +50,14 @@ class PointsSettingsViewController: UITableViewController {
             print("user point failed to save")
         }
     }
+
+    override func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        let textFieldText = NSString(string: textField.text!).replacingCharacters(in: range, with: string)
+        if textFieldText.isEmpty {
+            return true
+        } else if let integerValue = Int(textFieldText), integerValue <= INT_MAX {
```

Comment:
@khairul-alam-licon I guess you need to update the same in the Android sample app too. We should not limit the integer to 9 digits.